src/hydra/fetchJsonLd.ts

```diff
@@ -0,0 +1,50 @@
+const jsonLdMimeType = "application/ld+json";
+
+/**
+ * Sends a JSON-LD request to the API.
+ *
+ * @param {string} url
+ * @param {object} options
+ * @return {Promise.<object>} An object with a response key (the original HTTP response) and an optional body key (the parsed JSON-LD body)
```

Comment:
Fix `Promise.<object>` in `Promise<object>`, please

Reply:
I believe there is an argument to be made to remove the types entirely. I can't speak for other editors, but VS code will work with tsdoc annotations ootb.

See https://github.com/Microsoft/tsdoc.

Reply:
Let's remove useless docblock entirely then!